### PR TITLE
feat(module:transfer): use nzOneWay to make Transfer the one way style

### PR DESCRIPTION
**Pull request by OriginRing**
fix: #8643

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/NG-ZORRO/ng-zorro-antd/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Application (the showcase website) / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: components/transfer/transfer-list.component.ts [hunk 1]
```diff
@@ -161,11 +195,13 @@ export class NzTransferListComponent implements AfterViewInit {
   @Input() renderList: TemplateRef<RenderListContext> | null = null;
   @Input() render: TemplateRef<{ $implicit: TransferItem }> | null = null;
   @Input() footer: TemplateRef<{ $implicit: TransferDirection }> | null = null;
+  @Input() nzOneWay: boolean = false;
```
Comment (HyperLife1119) [inline components/transfer/transfer-list.component.ts:198]:
```suggestion
  @Input() oneWay: boolean = false;
```

Comment (OriginRing) [reply components/transfer/transfer-list.component.ts]: 已修改

**Review by HyperLife1119 — APPROVED** [review]
LGTM

File: components/transfer/demo/on-way.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import { Component, OnInit } from '@angular/core';
+
+import { TransferItem } from 'ng-zorro-antd/transfer';
+
+@Component({
+  selector: 'nz-demo-transfer-on-way',
```
Comment (Laffery) [inline components/transfer/demo/on-way.ts:6]: `on-way` -> `one-way`

File: components/transfer/transfer-list.component.ts [hunk 3]
```diff
@@ -137,7 +163,15 @@ import { NzTransferSearchComponent } from './transfer-search.component';
...
   },
-  imports: [NgClass, NzCheckboxModule, NgTemplateOutlet, NzEmptyModule, NzTransferSearchComponent],
+  imports: [
+    NgClass,
+    NzCheckboxModule,
+    NgTemplateOutlet,
+    NzEmptyModule,
+    NzTransferSearchComponent,
+    NzIconModule,
+    NzButtonComponent
```
Comment (Laffery) [inline components/transfer/transfer-list.component.ts:173]: Remove unused NzButtonModule

**Review by Laffery — CHANGES_REQUESTED** [review]

File: components/transfer/transfer-list.component.ts (hunk 3 above)
Comment (OriginRing) [reply components/transfer/transfer-list.component.ts]: Corrected

**Comment by HyperLife1119** [comment]
Please fix the test :)

**Review by Laffery — APPROVED** [review]
LGTM

**Pull request merged**